packages/shared/src/contexts/PaymentContext.tsx

```diff
@@ -189,23 +189,25 @@ export const PaymentContextProvider = ({
       }).value;
     }, [planTypes, productOptions]);
 
+  const giftOneYear: ProductOption = useMemo(
+    () =>
+      productOptions.find(
+        (option) => planTypes[option.value] === PlusPriceType.GiftOneYear,
+      ),
+    [planTypes, productOptions],
+  );
+
   const contextData = useMemo<PaymentContextData>(
     () => ({
       openCheckout,
       paddle,
-      productOptions,
+      productOptions: productOptions.filter(
+        ({ value }) => value !== giftOneYear?.value,
+      ),
       earlyAdopterPlanId,
-      oneTimePayment: {
-        // TODO: MI-756 - Replace with actual one-time payment plan
-        currencyCode: 'USD',
-        priceUnformatted: 88.88,
-        price: '$88.88',
-        label: 'One-time payment',
-        value: 'one-time',
-        extraLabel: '2 months free',
-      },
+      giftOneYear,
     }),
-    [earlyAdopterPlanId, openCheckout, paddle, productOptions],
+    [giftOneYear, earlyAdopterPlanId, openCheckout, paddle, productOptions],
```

Comment:
I think you need to merge upstream changes from `main` since the logic to disable purchasing for already plus members (that I did) might need adjusting for this.

Reply:
Basically there is a new `isPlusAvailable` value in context which probably needs to return `true` based on gift param